apps/foundation-demo/src/app/core/sidebar/containers/sidebar/sidebar.component.scss

```diff
@@ -1,17 +1,20 @@
 @import 'daffodil-util';
 
 .sidebar {
-  padding-top: 10px;
-  display: grid;
-  grid-template-rows: 100px repeat(4, 35px);
+  padding: 75px 25px 0 25px;
 
   &__close {
-    justify-self: end;
-    margin: 20px;
+    position: absolute;
+    top: 25px;
+    right: 25px;
     font-size: $medium-font-size;
     cursor: pointer;
-    line-height: 0;
-    height: 25px;
+  }
+
+  &__list {
+    padding: 0;
+    margin: 0;
+    list-style: none;
```

Comment:
Once you add the `daff-list` attribute, you shouldn't need some, if not all, of these styles.

Reply:
So, I closed this thinking it made sense to use the `sidebar__list` since the styles are different. I wonder though, if the sidebar needs `ul`/`li` here in the first place. I can't explicitly state a reason for needing it beyond simple "convention".

Reply:
I guess it doesn't need to be. the `ul/li` can be reserved for future use when/if there are flyout/sub-list items to the main list items.

Reply:
Do we need a ul and li then as well? I can't really see a use for it.

Reply:
I guess not. In this case, the footer links don't need to be `ul/li` either?

Reply:
I don't see why anything ever needs `ul/li`. There might be a reason. I've just never been curious what that reason could be.

Reply:
Just looked it up. Seems to be just to make HTML a bit more readable. So I'd say that because the footer is composed of attribute components that convey it is a list, there's no need to also use `ul/li`. But I don't really think it matters that much.

Reply:
Also, this is the second time we're making a list in foundation-demo. If we're already not using the `daff-list` component because the styling doesn't work, doesn't that mean it might be kind of useless?

Reply:
Looked it up and it's becoming more common to not use `ul/li` to increase readability, so I changed it to `div/span`.

And yes, `daff-list` is not needed unless we are thinking about lists for product descriptions, shipping information, etc. That would definitely be styled differently than what we have now since in that case, the bullets would remain.

Reply:
So, I'm not sure of this new API is any better. To me it seems just as verbose, merely swapping elements doesn't change much. I'd much prefer something like...

```
<daff-sidebar>
    <a daff-sidebar-item></a>
</daff-sidebar>
```